modules/tftp/tftp.rb

```diff
@@ -1,2 +1,10 @@
 require 'tftp/tftp_plugin'
-module Proxy::TFTP; end
+module Proxy::TFTP
+  class Capabilities
+
+    # check availability of isoinfo command
+    def self.has_extraction
+      system(which(Proxy::ArchiveExtract.SHELL_COMMAND)) ? 'extraction' : nil
```

Comment:
`system()` means you actually execute it. I think that's too much and would think it's sufficient to check it's there.

Reply:
Moved it directly to `tftp_plugin.rb`.